src/module-1/weirdString/index.js

```diff
@@ -1 +1,15 @@
-export const weirdString = (str = "") => {};
+export const weirdString = (str = "") => {
+	let result = "";
+	let arr = str.split(" ");
+	for (let i = 0; i < arr.length; i++) {
+		let part_1 = "";
+		let part_2 = "";
```

Comment:
you don't need to predefine this two variables. Values can be set immediately
`const part_1 = arr[i].slice(0, -1);
const part_2 = arr[i].slice(-1)`

Reply:
fixed